### PR TITLE
Travis mysql setup

### PR DESCRIPTION
**Pull request by rjt-gupta**

### CODE REVIEW

File: tanner/config.py [hunk 1]
```diff
@@ -21,6 +21,8 @@
                                         'php_code_injection': True, "crlf": True},
                    'SQLI': {'type': 'SQLITE', 'db_name': 'tanner_db', 'host': 'localhost', 'user': 'root',
                             'password': 'user_pass'},
+                   'MySQLI': {'type': 'MySQL', 'db_name': 'tanner_db', 'host': '127.0.0.1', 'user': 'root',
```
Comment (rnehra01) [inline tanner/config.py]: Don't change the code just to make tests. You should change the values in this `'SQLI': {'type': 'SQLITE', 'db_name': 'tanner_db', 'host': 'localhost', 'user': 'root',
                            'password': 'user_pass'}` for tests.

Comment (rjt-gupta) [reply tanner/config.py]: I thought it will be useful. No worries will keep that in mind :)

File: tanner/tests/test_config.py [hunk 2]
```diff
@@ -73,8 +73,8 @@ def test_get_when_file_dont_exists(self):
             'REDIS': {'host': 'localhost', 'port': 6379, 'poolsize': 80, 'timeout': 1},
             'EMULATORS': {'root_dir': '/opt/tanner'},
             'EMULATOR_ENABLED': {'sqli': True, 'rfi': True, 'lfi': True, 'xss': True, 'cmd_exec': True},
-            'SQLI': {'type': 'SQLITE', 'db_name': 'tanner_db', 'host': 'localhost', 'user': 'root',
-                     'password': 'user_pass'},
+            'SQLI': {'type': 'SQLITE', 'db_name': 'tanner_db', 'host': '127.0.0.1', 'user': 'root',
```
Comment (rnehra01) [inline tanner/tests/test_config.py]: Isn't it possible to leave it as it is and change the values of `host` and `password` in the `setUp` method?

Comment (rjt-gupta) [reply tanner/tests/test_config.py]: Yeah! we can hardcode the credentials in setup directly if we can?

Comment (rjt-gupta) [reply tanner/tests/test_config.py]: I don't think we can test db helpers without changing that. We can only do this for emulators tests.

Comment (rnehra01) [reply tanner/tests/test_config.py]: Look [here](https://stackoverflow.com/questions/30945870/how-to-mock-patch-a-class-imported-in-another-module), you can patch the `TannerConfig` class, see if it works.

**Comment by rjt-gupta** [comment]
I think let's merge the travis changes first and then we'll see the config setting in the test's PRs.

**Comment by rnehra01** [comment]
Add this change in that PR itself, it will be easy to track why was is it added.

**Comment by rjt-gupta** [comment]
> 
> 
> Add this change in that PR itself, it will be easy to track why was is it added.

Alright. Shall I close this PR?

**Pull request closed without merging**